package.json

```diff
@@ -1,28 +1,32 @@
 {
-  "name": "Brim",
+  "name": "brim",
```

Comment:
Just a mental note that we should double check this doesn't change the name of the app when we build a release version of it

Reply:
Yes, I have that same mental note.